pkg/database/import.go

```diff
@@ -0,0 +1,148 @@
+package database
+
+import (
+	"github.com/rotisserie/eris"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type experimentInfo struct {
+	sourceID int64
+	destID   int64
+}
+
+var experimentInfos = []experimentInfo{}
+
+// Import will copy the contents of input db to output db.
+func Import(input, output *DbInstance) error {
```

Comment:
Not clear what you are suggesting -- Go is very functional, yes?

Reply:
it is but like usually this is a main problem if Go language. If you open `Gorm` or `AWS` SDK or any other SDK it will be built using not a `functional` way, because in that case it will be hell of functions. So it will be nice to inject a bit of `object` oriented style here.

Reply:
i'll add a struct to hold db instances and the global var.